project/MiMa.scala

```diff
@@ -939,7 +939,7 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.actor.Deployer.lookup"),
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.util.WildcardTree.apply"),
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.util.WildcardTree.find"),
-        
+
```

Comment:
would be good to remove this diff, to avoid merge conflicts in other PR (this is a hot place)